src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-top: 46px;
+  }
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    font-size: inherit;
+    font-weight: inherit;
+    color: $main-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    @extend %small-text;
+    display: block;
+    margin: 0;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    @extend %small-text;
+    margin: 0;
+    color: $main-color;
+  }
+
+  &__stars {
+    display: flex;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-text {
+    margin: 0;
+    font-weight: 400;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__price-value {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    color: $main-color;
+  }
+
+  &__link {
+    display: block;
+    width: 166px;
+    height: 40px;
+    text-align: center;
+    background-color: $color-blue-accent;
+    color: #fff;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    line-height: 40px;
+    border: 1px solid rgba(0, 172, 220, 0);
+    &:hover {
```

add an empty line when you have nesting (check all over the page)
```suggestion
    border: 1px solid rgba(0, 172, 220, 0);
    
    &:hover {
```